RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingLanding/mobileAppointmentBookingLanding.js

```diff
@@ -543,38 +538,39 @@ export default class MobileAppointmentBookingLanding extends LightningElement {
     var currentDate = start;
     let currentDateArray = [];
 
-    if (this.currentAssignmentMethod == assignmentMethod.ASSIGN_TO_ME) {
+    if (this.currentAssignmentMethod === assignmentMethod.ASSIGN_TO_ME) {
       currentDateArray = this.dateArrayForQueryCurrentMobileWorkwerSlots;
     } else {
       currentDateArray = this.dateArrayForQueryAllMobilesWorkerSlots;
     }
 
     while (currentDate <= end) {
-      var addingDate = new Date(currentDate);
+      let addingDate = new Date(currentDate);
       currentDateArray.push(addingDate);
-      var tempDate = currentDate.setDate(currentDate.getDate() + 1);
+      let tempDate = currentDate.setDate(currentDate.getDate() + 1);
       currentDate = new Date(tempDate);
     }
 
     currentDateArray = Array.from(new Set(currentDateArray));
   }
 
   removeDatesFromCashArray() {
-    if (this.currentAssignmentMethod == assignmentMethod.ASSIGN_TO_ME) {
+    if (this.currentAssignmentMethod === assignmentMethod.ASSIGN_TO_ME) {
       this.dateArrayForQueryCurrentMobileWorkwerSlots = [];
     } else {
       this.dateArrayForQueryAllMobilesWorkerSlots = [];
     }
   }
 
   showAlertWithError(errorMessage) {
+    // eslint-disable-next-line no-alert
     alert(errorMessage);
   }
```

Comment:
Alerts don't work in LWCs on mobile..... so not sure why `showAlertWithError` exists.